ArduPlane/mode.cpp

```diff
@@ -88,9 +88,7 @@ bool Mode::enter()
 
 bool Mode::is_vtol_man_throttle() const
 {
-    if (!plane.quadplane.in_vtol_mode() &&
-        plane.quadplane.is_tailsitter() && 
-        plane.quadplane.tailsitter_transition_fw_complete() && 
+    if (plane.quadplane.in_tailsitter_fw_flight() &&
         plane.quadplane.assisted_flight) {
         // a tailsitter that has fully transisisoned to Q-assisted forward flight
```

Comment:
spelling error
and I think these comments could be improved by using complete sentences with punctuation
I hadn't realized this function was only for tailsitters; I may have misused it in the airmode PR

Reply:
Fixed spelling. Sentences and punctuation is not one of my strong points, I have improved it I hope. Yeah your right, changed to `is_tailsitter_in_fw_flight`

Reply:
The comments make sense to me now; thanks.